### ISSUE TITLE
close_mosaic() does not work

### ISSUE THREAD

**Issue by zhangming8** [Megvii-BaseDetection/YOLOX#85]
Hi, thanks for your great work.
   I find when using 'self.train_loader.close_mosaic()' mosaic augment will still be used, Is that right? 
  code: https://github.com/Megvii-BaseDetection/YOLOX/blob/main/yolox/core/trainer.py#L192

**Comment by Joker316701882** [comment]
No, after self.train_loader.close_mosaic() is called, both mosaic and mixup are disabled.

**Comment by Joker316701882** [comment]
This issue has been fixed.

**Comment by zhangming8** [comment]
So, in your paper, the change of AP curve in last 15 epochs is due to the L1 loss, not because the "closed" mosaic augment ?
